app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine-related errors."""
+    pass
```

Comment:
You should add a default message to your VaccineError. It could be something like 'There was an error related to the vaccine.'.